websites/S/Serendia Squad/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "BBQ",
+    "id": "622527559946731540"
+  },
+  "url": "serendiasquad.com",
+  "description": {
+    "tr": "Serendia Squad'ın kod paylaşım sitesi.",
+    "en": "Serendia Squad's code sharing website.",
+  },
+  "service": "Serendia Squad",
+  "version": "1.0.0",
+  "logo": "https://bbq.is-pretty.cool/5o6DS5b.png",
+  "thumbnail": "https://bbq.is-pretty.cool/5o6DS5b.png",
```

Comment:
Please change the thumbnail to a promotional image or a screenshot of the website